src/hooks/protections/receive_net_message.cpp

```diff
@@ -120,7 +120,7 @@ namespace big
 					if (g.session.log_chat_messages)
 						chat::log_chat(message, player, spam_reason, is_team);
 					g_notification_service.push("PROTECTIONS"_T.data(),
-                                      
+
```

Comment:
Unnecessary whitespace changes.

Reply:
Consequence of the formatter, those tabs shouldn't have been there to begin with.